README.md

```diff
@@ -57,42 +57,13 @@ Additionally you may find the following links useful:
 ## Samples
 - [Listing of all samples available in this repository ](./samples/README.md)
 
-## Tutorials
```

Comment:
Need to also update the table of contents at the top

Reply:
I have removed the tutorial links from the TOC and paragraphs.

Reply:
cool thanks! but shouldn't we replace it with a link to the new General information section?

Reply:
Somehow missed that. Just pushed a commit that adds the general information section to the TOC.